infrastructure/resource_group_level/modules/storage.bicep

```diff
@@ -0,0 +1,421 @@
+  parent: queueService
+  name: 'acbs-control-03'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource acbsWorkItemsQueue 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'acbs-workitems'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultDurableFunctionsHubControlQueue0 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'durablefunctionshub-control-00'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultDurableFunctionsHubControlQueue1 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'durablefunctionshub-control-01'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultDurableFunctionsHubControlQueue2 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'durablefunctionshub-control-02'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultDurableFunctionsHubControlQueue3 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'durablefunctionshub-control-03'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultDurableFunctionsHubWorkItemsQueue 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'durablefunctionshub-workitems'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultNumberGeneratorControlQueue0 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'numbergenerator-control-00'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultNumberGeneratorControlQueue1 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'numbergenerator-control-01'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultNumberGeneratorControlQueue2 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'numbergenerator-control-02'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultNumberGeneratorControlQueue3 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'numbergenerator-control-03'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource defaultNumberGeneratorWorkItemsQueue 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: queueService
+  name: 'numbergenerator-workitems'
+  properties: {
+    metadata: {}
+  }
+}
+
+resource acbsHistoryTable 'Microsoft.Storage/storageAccounts/tableServices/tables@2022-09-01' = {
+  parent: tableService
+  name: 'acbsHistory'
+  properties: {}
+}
+
+resource defaultAcbsInstancesTable 'Microsoft.Storage/storageAccounts/tableServices/tables@2022-09-01' = {
+  parent: tableService
+  name: 'acbsInstances'
+  properties: {}
+}
+
+// TODO:FN-417 Is this the right name?
```

Comment:
The date `202304` seems a bit transitory.

Reply:
@abhi-markan ?